### PR TITLE
[Hold for merge] Upgrade to DS-2.0.2

### PR DESCRIPTION
**Pull request by jillcary**
# Description and Motivation 
Upgrade Remote Monitoring to use the latest version of Device Simulation.

The PR for local deployments can be found here:
https://github.com/Azure/remote-monitoring-services-dotnet/pull/209

Hold merge for the Simulation template to be added.
https://github.com/Azure/device-simulation-dotnet/pull/330

# Change type 

- [ ] Bug fix
- [ ] New feature
- [x] Enhancement
- [ ] Breaking change (breaks backward compatibility)

**Checklist:**

- [x] All tests passed
- [x] The code follows the code style and conventions of this project
- [x] The change requires a change to the documentation
- [x] I have updated the documentation accordingly

---
This change is [Reviewable](https://reviewable.io/reviews/azure/pcs-cli/446)

### CODE REVIEW

File: solutions/remotemonitoring/scripts/all-in-one.yaml [hunk 1]
```diff
@@ -197,46 +197,101 @@ spec:
...
-          valueFrom:
-            configMapKeyRef:
-              name: deployment-configmap
-              key: security.cors.whitelist
-        - name: PCS_APPLICATION_SECRET
-          valueFrom:
-            configMapKeyRef:
-              name: deployment-configmap
-              key: security.application.secret
+        - name: device-simulation-pod
```
Comment (zhang-hua) [inline solutions/remotemonitoring/scripts/all-in-one.yaml]: The indents looks not aligned correctly. Can you check that?

File: solutions/remotemonitoring/single-vm/docker-compose.java.yml [hunk 2]
```diff
@@ -69,23 +69,39 @@ services:
...
-    # How one could mount custom device models
-    #volumes:
-    #  - ./my-device-models:/app/data:ro
+      - PCS_SUBSCRIPTION_ID
+      - PCS_WEBUI_AUTH_AAD_APPID=${PCS_AAD_APPID}
+      - PCS_WEBUI_AUTH_AAD_TENANT=${PCS_AAD_TENANT}
+      - PCS_AAD_CLIENT_SP_ID=${PCS_AAD_APPID}
+      - PCS_AAD_SECRET=${PCS_AAD_APPSECRET}
+      - PCS_RESOURCE_GROUP=${PCS_SOLUTION_NAME}
+      - PCS_IOHUB_NAME=${PCS_IOTHUB_NAME}
```
Comment (zhang-hua) [inline solutions/remotemonitoring/single-vm/docker-compose.java.yml]: nit: is it a typo `PCS_IOHUB_NAME`?

Comment (jillcary) [reply solutions/remotemonitoring/single-vm/docker-compose.java.yml]: This is intentional as the simulation service has a typo in the environment variable. See here:
https://github.com/Azure/device-simulation-dotnet/blob/0ced7244642ce2872ff66cc395fa8c60d0b90eef/WebService/appsettings.ini#L213

File: solutions/remotemonitoring/scripts/all-in-one.yaml [hunk 3]
```diff
@@ -198,7 +198,7 @@ spec:
     spec:
       containers:
       - name: device-simulation-pod
-        image: azureiotpcs/device-simulation-dotnet:DS-1.0.2
+        image: azureiotpcs/device-simulation-dotnet:DS-2.0.1
```
Comment (hathind-ms) [inline solutions/remotemonitoring/scripts/all-in-one.yaml]: next release version will be DS-2.0.2

Comment (jillcary) [reply solutions/remotemonitoring/scripts/all-in-one.yaml]: Updated.

**Comment by dluc** [comment]
Someone should fix the "IOHUB" typo rather than carrying it over

**Pull request merged**